backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/internal/BpkCalendarDayCell.kt

```diff
@@ -70,15 +71,20 @@ internal fun BpkCalendarDayCell(
         horizontalAlignment = Alignment.CenterHorizontally,
         modifier = modifier
             .padding(bottom = BpkSpacing.Lg)
-            .selectable(
+            .clickable(
                 indication = null,
-                selected = selection != null,
+//                selected = selection != null,
                 enabled = !inactive,
                 onClick = { onClick(model) },
+                onClickLabel = model.onClickLabel, // this is announced as the action to take when clicking (ie double tap to select end date)
```

Comment:
we can probably remove the comments now :)

Reply:
Done!